cmd/clusterctl/main_integration_test.go

```diff
@@ -39,7 +39,7 @@ const (
 
 // run these tests with the flag "-update" to update the values stored in all of the golden files
 var (
-	update = flag.Bool("update", false, "update .golden files")
+	update = flag.Bool("update", true, "update .golden files")
```

Comment:
This should be reverted before merging :)

Reply:
Ohhhhhh I get it now